app/models/intake.rb

```diff
@@ -572,10 +576,11 @@ def formatted_mailing_address
     ADDRESS
   end
 
-  def create_13614c_document(filename)
+  def create_intake_document(filename)
+    filename ||= "2020 13614-C with 15080.pdf"
```

Comment:
Ruby only seems to use the default value _if no value is passed in at all_ for that parameter. That means that passing in an explicit "nil" value into this method would cause the default filename to get overwritten if i did 

`def create_intake_document(filename='2020...')`